diff --git a/drivers/media/usb/uvc/uvcvideo.h.rej b/drivers/media/usb/uvc/uvcvideo.h.rej
deleted file mode 100644
index 19c2b5d4dda7..000000000000
--- a/drivers/media/usb/uvc/uvcvideo.h.rej
+++ /dev/null
@@ -1,10 +0,0 @@
---- drivers/media/usb/uvc/uvcvideo.h
-+++ drivers/media/usb/uvc/uvcvideo.h
-@@ -222,6 +222,7 @@
- #define UVC_QUIRK_RESTRICT_FRAME_RATE	0x00000200
- #define UVC_QUIRK_RESTORE_CTRLS_ON_INIT	0x00000400
- #define UVC_QUIRK_FORCE_Y8		0x00000800
-+#define UVC_QUIRK_APPEND_UVC_HEADER	0x00001000
- 
- /* Format flags */
- #define UVC_FMT_FLAG_COMPRESSED		0x00000001

diff --git a/firmware/tegra21x_xusb_firmware b/firmware/tegra21x_xusb_firmware
deleted file mode 100644
index 9930bfa7edd0..000000000000
Binary files a/firmware/tegra21x_xusb_firmware and /dev/null differ

